projects/packages/my-jetpack/_inc/data/utils/get-my-jetpack-window-state.ts

```diff
@@ -1,17 +1,37 @@
-type MyJetpackState = Window[ 'myJetpackInitialState' ];
+type InitialState = Window[ 'myJetpackInitialState' ];
+type RestState = Window[ 'myJetpackRest' ];
+type ConnectionState = Window[ 'JP_CONNECTION_INITIAL_STATE' ];
```

Comment:
`InitialState` nomenclature is weird, but I guess it's better to be consistent with backend 🤷‍♂️